src/plugins/dashboard/server/dashboard_saved_object/dashboard_saved_object.ts

```diff
@@ -38,6 +40,33 @@ export const createDashboardSavedObjectType = ({
       };
     },
   },
+  modelVersions: {
```

Comment:
We could specify the `switchToModelVersionAt` top-level property. It would have to be `8.10.0`, because it can not be higher than that. But not really sure if it is necessary to include anymore.

Reply:
I removed this per @jloleysens's suggestion, since `8.10.0` is the default value :+1: 

https://github.com/elastic/kibana/pull/174714#discussion_r1514149732